Hyperbolic_triangulation_2/examples/Hyperbolic_triangulation_2/CMakeLists.txt

```diff
@@ -6,6 +6,10 @@ project( Hyperbolic_triangulation_2_Examples )
 
 cmake_minimum_required(VERSION 2.8.10)
 
+if(POLICY CMP0054)
+  cmake_policy(SET CMP0054 NEW)
+endif()
+
```

Comment:
I disagree with this patch. `CMP0054` was introduced by CMake 3.1, that we require.

The right patch is to fix the `cmake_minimum_required` declaration.

Reply:
Use `cmake_minimum_required(VERSION 3.1...3.13)` everywhere.